pymtl3/passes/backends/verilog/VerilogPlaceholderPass.py

```diff
@@ -410,7 +410,13 @@ def _get_auto_prefix( s, m ):
     if 'placeholder' in pymtl_config and \
        'auto_prefix' in pymtl_config['placeholder'] and \
        pymtl_config.getboolean( 'placeholder', 'auto_prefix' ):
-      return f"{os.path.basename(parent_dir)}_"
+      if cwd == parent_dir:
```

Comment:
I updated the way the placeholder pass generates auto-prefix -- does this match your intentions?

Reply:
I think yes. Why was it broken before you fixed it?

Reply:
If there's a `pymtl.ini` under `parent_dir` it would just return a single underscore -- the expected behavior would be to return the base name of `parent_dir`.